tox.ini

```diff
@@ -0,0 +1,94 @@
+; a generative tox configuration, see: https://tox.readthedocs.io/en/latest/config.html#generative-envlist
+
+[tox]
+envlist =
+    clean,
+    docs,
+    {py35,py36,py37,pypy,pypy3},
+    flake8,
+    report
+
+[testenv]
+basepython =
+    pypy: {env:TOXPYTHON:pypy}
+    pypy3: {env:TOXPYTHON:pypy3}
+    py35: {env:TOXPYTHON:python3.5}
+    {py36,docs,spell}: {env:TOXPYTHON:python3.6}
+    py37: {env:TOXPYTHON:python3.7}
+    {clean,check,report,codecov}: {env:TOXPYTHON:python3}
+setenv =
+    PYTHONPATH={toxinidir}/tests
+    PYTHONUNBUFFERED=yes
+passenv =
+    *
+usedevelop = false
+deps =
```

Comment:
Isn't it enough with installing poliastro and let `pip` + `install_requires` in `setup.cfg` do the rest?

Reply:
That is what I thought when coding this. However, and as far as I read tox.ini looks for the requirements.txt file in order to install all the dependencies. It is kind of redundant to have a requirements.txt and a setup.py or setup.cfg with the same install_requires.

I will look for more information on the topic to try solving this.

Reply:
Tox i s now intalling the dependencies from setup.cfg, so everything is right. Probably was because I let the `deps = `. So this is solved.